x-pack/plugins/ml/public/application/components/controls/select_severity/select_severity.tsx

```diff
@@ -102,7 +102,7 @@ export const getSeverityOptions = () =>
         </EuiHealth>
         <EuiSpacer size="xs" />
         <EuiText size="xs" color="subdued">
-          <p className="euiTextColor--subdued">
+          <p>
```

Comment:
While testing this, I noticed that there is no longer any padding between these controls and the anomalies table in the ML Anomaly Explorer. Has something in this upgrade which could cause a change to the padding around the flex items - relevant code is in https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/public/application/explorer/explorer.tsx#L558

Before:
![image](https://user-images.githubusercontent.com/7405507/173862132-2c094944-7eb5-4e08-976d-c36546288da2.png)


After:
![anomalies_table_selects](https://user-images.githubusercontent.com/7405507/173862021-10ee477a-bfe4-4c29-a76c-bf7de57f2184.png)

Reply:
It's probably related to this line specifically: https://github.com/elastic/kibana/blob/9fc6b7de04d6e379da9675bc0987fc5820b5cf3b/x-pack/plugins/ml/public/application/explorer/explorer.tsx#L574

The plain `.euiText` class no longer has styles associated with it. You should swap to using `<EuiText />` directly or determine if that wrapper is necessary at all.